jenkins-pipelines/upgrade-major-scylla-k8s-gke.jenkinsfile

```diff
@@ -0,0 +1,11 @@
+#!groovy
+
+def lib = library identifier: 'sct@snapshot', retriever: legacySCM(scm)
+
+rollingOperatorUpgradePipeline(
+    backend: 'k8s-gke',
+    base_versions: '["4.1.5"]',
+    new_version: '4.2.0',
```

Comment:
Is there any reason to not use '4.2.1' here?

Reply:
switched to 4.2.1